### PR TITLE
Task to manage repositories for migration

### PR DESCRIPTION
**Pull request by curt-matthews**

### CODE REVIEW

**Comment by henrygrover** [comment]
Looks great, nice work guys

**Comment by itewk** [comment]
@curt-matthews i rebased your branch on master and did a force push so that it is elligable for merge:

```
git checkout master
git pull origin master
git checkout curt-matthews
git rebase origin/master
git push origin curt-matthews --force # since branch was re-based history changed
```

File: README.md [hunk 1]
```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
 * Disks have already been resized.
 * Environments are not utilizing database replication.
 
+#### Additionaly required playbooks
+* cfme-repositories.yml
```
Comment (itewk) [inline README.md:16]: this isn't a playbook its a task list and since it's included as part of this repository i would not explicitly call it out as a dependency since there is nothing the user of this repository would need to do.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
...
+
 #### Required groups
 * cfme
 * cfme-appliancees
 * cfme-databases
+
+### cfme-repositories.yml
```
Comment (itewk) [inline README.md:23]: This isn't a playbook, just a task list, so no need to list since it isn't a "API".

File: README.md [hunk 3]
```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
...
+Disables all currently configured repositories and enables the required playbooks from migrate-5.7-to-5.8.yml.
+
+### configure_noop_vmware_rhel_vms.yml
+Configures noop for the queue scheduler for RHEL VMs on VMware infrastructure per [How to use the Noop IO Scheduler](https://access.redhat.com/solutions/109223).
```
Comment (itewk) [inline README.md:27]: Since this isn't specific to CFME should this playbook live in https://github.com/rhtconsulting/ansible-cloud_utils instead?

File: playbooks/configure_noop_vmware_rhel_vms.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          dest: /etc/grub.cfg
+          regexp: '^kernel(.*)'
+          line: 'kernel \1 elevator=noop'
+          state: present
+          backrefs: True
+          backup: True
+        when:
+          - "not original_grub_config['content'] | b64decode | regex_search('(elevator=noop)')"
+      
+      when:
```
Comment (itewk) [inline playbooks/configure_noop_vmware_rhel_vms.yml:56]: Since this is only needed/recomended on VMware should there be a condition for that? Example: https://github.com/rhtconsulting/ansible-cloud_utils/blob/master/playbooks/configure_vm_network_and_ip.yml#L22

File: playbooks/configure_noop_vmware_rhel_vms.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+          backup: True
+        when:
+          - "not original_grub2_config['content'] | b64decode | regex_search('(elevator=noop)')"
+        register: grub2_changed
+
+      - name: Rebuild the /boot/grub2/grub.cfg
+        command: /usr/sbin/grub2-mkconfig -o /boot/grub2/grub.cfg
+        when:  grub2_changed|changed
+
+      when:
```
Comment (itewk) [inline playbooks/configure_noop_vmware_rhel_vms.yml:33]: Since this is only needed/recomended on VMware should there be a condition for that? Example: https://github.com/rhtconsulting/ansible-cloud_utils/blob/master/playbooks/configure_vm_network_and_ip.yml#L22

File: playbooks/tasks [hunk 6]
```diff
@@ -0,0 +1 @@
+../tasks
```
Comment (itewk) [inline playbooks/tasks:1]: this is unrelated to the new playbook and is rather a fix for an existing playbook so maybe just submit as a separate pull request.

File: tasks/cfme-repositories.yml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+- name: CFME | Disable All Repositories
```
Comment (itewk) [inline tasks/cfme-repositories.yml:1]: this is unrelated to the new playbook and is rather a fix for an existing playbook so maybe just submit as a separate pull request.

**Review by itewk — CHANGES_REQUESTED** [review]

**Comment by itewk** [comment]
@curt-matthews to pull in the rebased branch:

```
git checkout curt-matthews
git origin fetch
git rebase origin/curt-matthews
```

**Comment by itewk** [comment]
merged part of this in as part of https://github.com/rhtconsulting/ansible-redhat_cloudforms/pull/3
